### PR TITLE
only load dags in dagbag that have been migrated

### PR DESCRIPTION
**Pull request by pvditt**
This is a work around for the poor regex negative look back performance to only load migrated DAGs in the Airflow 2 cluster.

Related PR: https://github.com/lyft/airflowinfra2/pull/109

### CODE REVIEW

**Review by ArgentFalcon — APPROVED** [review]

File: setup.py
Comment (eschachar) [inline setup.py:48]: Have you released this version of Airflow, or do you still need to do that?

Comment (pvditt) [reply setup.py:48]: was holding off on that until there was merged

**Pull request merged**
